tests/test_algorithm.py

```diff
@@ -1156,14 +1249,14 @@ def test_set_max_leverage(self):
 
         # Set max leverage to 0 so buying one share fails.
         def handle_data(algo, data):
-            algo.order(self.sid, 1)
+            algo.order(algo.sid(self.sidint), 1)
```

Comment:
Line 1260 had a conflict when self.sid and algo.sid mapped to the sid() method, rather than the field sid, so I go rid of the conflict by naming the integer sidint.